portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -92,4 +106,18 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
+
+  /**
+   * Inner class for Comments.
+   */
+  class Comment {
+    private String text;
+    private Date date;
```

Comment:
it's good practice to add a comment for fields, in case there is any ambiguity about what the field does/holds

Reply:
Ah I didn't know that, thanks for the heads up -- just updated this.